src/Commands/Reuniao.ts

```diff
@@ -0,0 +1,52 @@
+import Command from "@core/Contracts/Command"
+import Context from "@core/Contracts/Context"
+import InvalidArgsException from "@core/Exceptions/InvalidArgs"
+
+export default class Reuniao extends Command {
+  public get description() {
+    return "Coloca o servidor em modo reunião."
+  }
+
+  public get roles(): string[] {
+    return [process.env.ADMIN_ROLE!]
+  }
+
+  public get roleValidationMessages() {
+    return {
+      [process.env
+        .ADMIN_ROLE!]: "Apenas administradores podem usar esse comando"
+    }
+  }
+
+  public help(): string {
+    return ":x: Como usar: `!reuniao <on|off>`"
+  }
+
+  public validate({ arg }: Context): void | never {
+    const states = {
+      on: 1,
+      off: 2
+    }
+
+    if (!(arg in states)) {
+      throw new InvalidArgsException(this.help())
+    }
+  }
+
+  public async run({ client, arg }: Context): Promise<void> {
+    client.channels
+      .filter(
+        (channel) =>
+          !/Reunião/i.test((channel as any).name) &&
+          !/My Bot Server/i.test((channel as any).name)
```

Comment:
as @kjkGustavo pointed out at live, I think that using the ids here is a better idea, also it should be placed on a settings file and not be hard coded, I'm sure that we will delete the channel by accident sometime in the future.